src/transforms/push.ts

```diff
@@ -58,12 +59,14 @@ export function createJobData(payload, jiraHost: string) {
 		// Creates an array of shas for the job processor to work on
 		shas.push({ id: commit.id, issueKeys });
 	}
+
 	return {
 		repository,
 		shas,
 		jiraHost,
 		installationId: payload.installation.id,
-		webhookId: payload.webhookId || "none"
+		webhookId: payload.webhookId || "none",
+		webhookReceived: payload.webhookReceived || "none"
```

Comment:
If webhookReceived is Date then defaulting it to "none" isn't good hence it is a "string" data type. 

@mboudreau  What would you recommend? Would it be OK to have undefined here? If we serialise it and put on redis queue would it be still undefined when we deserialise?

Reply:
Changed it to undefined. Added checks to see if `webhookReceived` exists before calling `calculateProcessingTimeInSeconds`